.github/workflows/generate_sbom.py

```diff
@@ -0,0 +1,20 @@
+from json import dumps
+
+from requests import get
+
+with open("opentelemetry-python.spdx.json", "w") as sbom_file:
```

Comment:
I took a look at the actual API response

1. It says
   > Exact versions could not be resolved for some packages. For more information: https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#dependencies-included.
2. The relationships in the file look like
   ```json
      { 
        "relationshipType": "DEPENDS_ON",
        "spdxElementId": "SPDXRef-com.github.open-telemetry-opentelemetry-python",
        "relatedSpdxElement": "SPDXRef-pip-asgiref-3.7.2"
      }
      ```
      My reading of this is "the opetnelemetry-python repo depends on X" rather than linking individual artifacts (e.g. opentelemetry-api) to their dependencies. I'm not sure how helpful that is for consumers of the SBOM.

Reply:
> I took a look at the actual API response
> 
> 1. It says
>    > Exact versions could not be resolved for some packages. For more information: [docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#dependencies-included](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#dependencies-included).

Right, I looked into the SBOM file, found several packages with an empty value for `versionInfo`, here they are:

```
{
    "SPDXID": "SPDXRef-com.github.open-telemetry-opentelemetry-python",
    "name": "com.github.open-telemetry/opentelemetry-python",
    "versionInfo": "",
    "downloadLocation": "git+https://github.com/open-telemetry/opentelemetry-python",
    "licenseDeclared": "Apache-2.0",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION",
    "externalRefs": [
        {
            "referenceCategory": "PACKAGE-MANAGER",
            "referenceType": "purl",
            "referenceLocator": "pkg:github/open-telemetry/opentelemetry-python"
        }
    ]
}
{
    "SPDXID": "SPDXRef-pip-opencensus",
    "name": "pip:opencensus",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opencensus-ext-flask",
    "name": "pip:opencensus-ext-flask",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opentelemetry-api",
    "name": "pip:opentelemetry-api",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opentelemetry-exporter-jaeger",
    "name": "pip:opentelemetry-exporter-jaeger",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opentelemetry-instrumentation-sqlite3",
    "name": "pip:opentelemetry-instrumentation-sqlite3",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opentelemetry-opencensus-shim",
    "name": "pip:opentelemetry-opencensus-shim",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opentelemetry-sdk",
    "name": "pip:opentelemetry-sdk",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-opentelemetry-opentracing-shim",
    "name": "pip:opentelemetry-opentracing-shim",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-redis",
    "name": "pip:redis",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-redis-opentracing",
    "name": "pip:redis-opentracing",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-googleapis-common-protos",
    "name": "pip:googleapis-common-protos",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-requests",
    "name": "pip:requests",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
{
    "SPDXID": "SPDXRef-pip-asgiref",
    "name": "pip:asgiref",
    "versionInfo": "",
    "downloadLocation": "NOASSERTION",
    "filesAnalyzed": false,
    "supplier": "NOASSERTION"
}
```